### PR TITLE
Add browser support to README | Fixes #888

### LINKED ISSUE #888: Update Documentation with Browser Support Matrix
Browser support matrix.

### PR DESCRIPTION
**Pull request by carloslancha**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,6 +21,7 @@
 
 3. Install project dependencies:
 
+  ```  yarn install
```
Comment (matuzalemsteles) [inline README.md]: hey @carloslancha, This is breaking the code blocks below.

Comment (carloslancha) [reply README.md]: Don't know what's that doing there, thx!

**Comment by carloslancha** [comment]
@matuzalemsteles fixed

**Comment by matuzalemsteles** [comment]
Merged, thanks @carloslancha .

**Pull request merged**